runelite-client/src/main/java/net/runelite/client/plugins/eeng1n/woodcutting/threetickteaks/ThreeTickTeaks.java

```diff
@@ -0,0 +1,164 @@
+package net.runelite.client.plugins.eeng1n.woodcutting.threetickteaks;
+
+
+import com.google.inject.Provides;
+import lombok.extern.slf4j.Slf4j;
+import net.runelite.api.*;
+import net.runelite.api.events.GameTick;
+import net.runelite.api.widgets.Widget;
+import net.runelite.client.Notifier;
+import net.runelite.client.callback.ClientThread;
+import net.runelite.client.config.ConfigManager;
+import net.runelite.client.eventbus.Subscribe;
+import net.runelite.client.plugins.Plugin;
+import net.runelite.client.plugins.PluginDescriptor;
+import net.runelite.client.plugins.microbot.Microbot;
+import net.runelite.client.plugins.microbot.util.gameobject.Rs2GameObject;
+import net.runelite.client.plugins.microbot.util.inventory.Inventory;
+import net.runelite.client.plugins.microbot.util.inventory.Rs2Inventory;
+import net.runelite.client.plugins.microbot.util.menu.Rs2Menu;
+import net.runelite.client.plugins.microbot.util.mouse.VirtualMouse;
+import net.runelite.client.plugins.microbot.util.npc.Rs2Npc;
+import net.runelite.client.ui.overlay.OverlayManager;
+
+import javax.inject.Inject;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+
+import static net.runelite.client.plugins.microbot.util.Global.sleep;
+import static net.runelite.client.plugins.microbot.util.Global.sleepUntilOnClientThread;
+import static net.runelite.client.plugins.natepainthelper.Info.*;
+
+@PluginDescriptor(
+        name = PluginDescriptor.Engin + " 3T Teaks",
+        description = "Performs 3T Teaks",
+        enabledByDefault = false
+)
+@Slf4j
+public class ThreeTickTeaks extends Plugin {
+    @Inject
+    private Client client;
```

Comment:
The following injects can be removed
-client
-clientThread
-notifier

these are injected and set in the MicrobotPlugin.java